src/server/new-request-handlers/commits.js

```diff
@@ -1,72 +1,28 @@
 /*jshint loopfunc: true */
 "use strict";
 
-var pool = require('../db/index.js');
+var users_dashboards = require('../queries/users_dashboards.js');
+var diffs = require('../queries/diffs.js');
 
 module.exports = {
   handlePost: function (req, res, next) {
-    var dashboards_id = req.body.dashboards_id;
+    var dashboards_signature_hash = req.body.dashboards_signature_hash;
     var diffs = req.body.diffs;
-    var last_pulled_commit = req.body.last_pulled_commit;
-    var users_id = req.body.users_id;
+    var new_params = {
```

Comment:
I've been using camelcase for my variable names, except for actual db field names, or property names for information sent between client/server (when the property directly corresponds to a field in the db). so I would recommend newParams over new_params. But not super important.